layers/meta-tegrademo/recipes-bsp/u-boot/u-boot-tegra_%.bbappend

```diff
@@ -0,0 +1,10 @@
+FILESEXTRAPATHS_prepend := "${THISDIR}/${PN}:"
+
+SRC_URI_append = " \
+  file://0014-common-bootm.c-Increase-CONFIG_SYS_BOOTM_LEN-to-64MB.patch \
+  file://0016-tools-fit_check_sign.c-Update-usage-function.-Add-c-.patch \
+  file://t210-Enable-FIT-image-support.cfg \
```

Comment:
Do you want this here, or below (i.e., for `tegra210` only)?

Reply:
For the current implementation yes only for `Jetson Nano` when I'm able to validate all stuff working fine, I will add the `Jetson TX2`